libcodechecker/server/api/report_server.py

```diff
@@ -2245,9 +2251,12 @@ def massStoreRun(self, name, tag, version, b64zip, force,
 
                 run_history_time = datetime.now()
 
-                check_commands, check_durations = \
+                check_commands, check_durations, skip_file_lines = \
                     store_handler.metadata_info(metadata_file)
 
+                skiphandler = skiplist_handler.SkipListHandler("")
```

Comment:
It is not an elegant way to pass an empty string. Instead of this we can handle the case when no parameter is given for this function.